src/core/server/integration_tests/saved_objects/migrations/group5/pickup_updated_types_only.test.ts

```diff
@@ -68,18 +66,18 @@ describe('pickupUpdatedMappings', () => {
 
       // we tamper the baseline mappings to simulate some root fields changes
       const baselineMappings = await client.indices.getMapping({ index: defaultKibanaIndex });
-      const _meta = baselineMappings[`${defaultKibanaIndex}_${currentVersion}_001`].mappings
-        ._meta as IndexMappingMeta;
-      _meta.migrationMappingPropertyHashes!.namespace =
-        _meta.migrationMappingPropertyHashes!.namespace + '_tampered';
-      await client.indices.putMapping({ index: defaultKibanaIndex, _meta });
+      const properties = Object.values(baselineMappings)[0].mappings.properties!;
+      (properties.references as any).properties.description = {
+        type: 'text',
+      };
+      await client.indices.putMapping({ index: defaultKibanaIndex, properties });
 
       await runMigrations();
 
       const logs = await parseLogFile(defaultLogFilePath);
 
       expect(logs).toContainLogEntry(
-        'Kibana is performing a compatible upgrade and the mappings of some root fields have been changed. For Elasticsearch to pickup these mappings, all saved objects need to be updated. Updated root fields: namespace.'
```

Comment:
NIT: so even before this PR, we were only returning the information about the first mapping that changed. Not that relevant, but why so?

Reply:
Actually we're logging a comma separated list here. The test happens to only change one.